opentracing/src/main/java/io/opentracing/Span.java

```diff
@@ -26,7 +28,8 @@
    * <p>This should be the last call made to any span instance, and to do otherwise leads to
    * undefined behavior.
    */
-  void finish();
```

Comment:
You can't remove this method as it implements the [Specification](http://opentracing.io/spec/#the-span-interface).

`close()` can delegate to it though.